core/storage/trie/polkadot_trie/polkadot_trie_impl.cpp

```diff
@@ -60,18 +62,24 @@ namespace kagome::storage::trie {
     return outcome::success();
   }
 
+  outcome::result<void> detachNode(PolkadotTrie::NodePtr &,
```

Comment:
I perfectly understand you are solving compiling issues by putting method declaration here. 
Since it is here, why not put the whole definition here too, or at least leave the parameters name for readability purposes?

Reply:
I wanted to leave declaration order the same to have more readability of changes. But as you both agree, let's move declarations.